src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindOperation.java

```diff
@@ -55,6 +58,22 @@ public Uni<FindResponse> getDocuments(QueryExecutor queryExecutor) {
     }
   }
 
+  @Override
+  public ReadDocument getEmptyDocuments() {
+    ObjectNode rootNode = objectMapper().createObjectNode();
+    DocumentId documentId = null;
+    for (DBFilterBase filter : filters) {
+      if (filter instanceof DBFilterBase.IDFilter) {
+        documentId = ((DBFilterBase.IDFilter) filter).value;
+        JsonNode id = ((DBFilterBase.IDFilter) filter).value.asJson(objectMapper());
+        rootNode.putIfAbsent("_id", id);
```

Comment:
This probably works fine, given that `rootNode` is empty Document: we have to ensure `_id` is always the first property.

Reply:
The document from getEmptyDocuments() is used in ReadAndUpdateOperation, where updates part of update clause are applied and Shredder.shred() is called. Shredder takes care of keeping the "_id" as the first field.